documentation/how-to/optimize-deployment-performance.md

```diff
@@ -0,0 +1,42 @@
+# Optimise your deployment performance
+This guide describes steps you can take to optimize the performance of your Superset deployment.
```

Comment:
I notice you're torn between the use of optimise and optimize :D

Reply:
optimise - we follow British English.